app/datatables/effective_tags_datatable.rb

```diff
@@ -0,0 +1,16 @@
+class EffectiveTagsDatatable < Effective::Datatable
```

Comment:
Both these datatables should be in the Admin namespace

Could you rename these to

`app/datatables/admin/effective_tags_datatable.rb`

If they're in the admin namespace, all the actions_col will "just work" as per the routes